packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -175,7 +194,18 @@ export default {
   computed: {
     extensions () {
       // These are the default extensions needed just for plain text.
-      const extensions = [CodeBlock, Document, Paragraph, Text];
+      const extensions = [
+        Blockquote,
+        Bold,
+        BulletList,
+        Document,
+        Italic,
+        ListItem,
+        Paragraph,
+        Strike,
+        Text,
+        Underline,
+      ];
       if (this.link) {
         extensions.push(this.getExtension(Link, this.link));
       }
```

Comment:
Just a small improvement suggestion for this part. We intended the editor's capabilities to have an opt-in structure, like we did with the link extension, so for example you need to explicitly pass "italic" as a prop to enable the italic extension. This is so that we can control which extensions are actually active in the editor as I _think_ currently all of these are all active, but we might not actually want them in every place we use the editor. AFAIR even without the toolbar UI you can use cmd+B to bold text for example. @emariotti-dialpad